src/client/interpreter/configuration/interpreterSelector/commands/installPython.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+
+import { inject, injectable } from 'inversify';
+import { IExtensionSingleActivationService } from '../../../../activation/types';
+import { ExtensionContextKey } from '../../../../common/application/contextKeys';
+import { ICommandManager, IContextKeyManager } from '../../../../common/application/types';
+import { PythonWelcome } from '../../../../common/application/walkThroughs';
+import { Commands, PVSC_EXTENSION_ID } from '../../../../common/constants';
+import { IBrowserService, IDisposableRegistry } from '../../../../common/types';
+import { IPlatformService } from '../../../../common/platform/types';
+
+@injectable()
+export class InstallPythonCommand implements IExtensionSingleActivationService {
+    public readonly supportedWorkspaceTypes = { untrustedWorkspace: false, virtualWorkspace: false };
+
+    constructor(
+        @inject(ICommandManager) private readonly commandManager: ICommandManager,
+        @inject(IContextKeyManager) private readonly contextManager: IContextKeyManager,
+        @inject(IBrowserService) private readonly browserService: IBrowserService,
+        @inject(IPlatformService) private readonly platformService: IPlatformService,
+        @inject(IDisposableRegistry) private readonly disposables: IDisposableRegistry,
+    ) {}
+
+    public async activate(): Promise<void> {
+        this.disposables.push(this.commandManager.registerCommand(Commands.InstallPython, () => this._installPython()));
+    }
+
+    public async _installPython(): Promise<void> {
+        if (this.platformService.isWindows) {
+            const version = await this.platformService.getVersion();
+            if (version.major !== 8) {
```

Comment:
```suggestion
            if (version.major > 8) {
```
What if someone opens this on lower version of windows?

Reply:
Actually VSCode doesn't support a lower version: https://code.visualstudio.com/docs/supporting/requirements#_platforms